tests/integration/create_token.py

```diff
@@ -59,7 +59,7 @@
             "user_id": "integration-test",
             "period_str": "April 2016",
             "period_id": "201604",
-            "ru_ref": "123456789012A",
+            "ru_ref": "12346789012A",
```

Comment:
If the Payload should mirror what we're expecting the output to be, should the check letter be removed from this instead of at the time of doing the assertion?